circuitpython_library_pypi_stats.md

```diff
@@ -1,10 +1,9 @@
 # Adafruit CircuitPython Library Download Stats
-![Blinka With Tux](https://cdn-learn.adafruit.com/assets/assets/000/089/982/medium640/raspberry_pi_main_image.jpg)  
+![Blinka With Tux](https://raw.githubusercontent.com/tekktrik/adabot/dev/update-file-bundle/assets/BlinkaTux.jpg)  
```

Comment:
Is this temporary while you await the updated Blinkas?

Reply:
Ah, no, good catch, it should be `adafruit`.